src/middlewared/middlewared/etc_files/local/nginx/nginx.conf

```diff
@@ -168,6 +171,14 @@ http {
             alias /usr/local/www/webui;
         }
 
+        location /stub_status {
```

Comment:
How is this used? and why is it not within /netdata location?

Reply:
Nginx exposes this separately and any service can make use of this if they want to get stats for nginx. Netdata uses this to display nginx respective stats. So this is not binded to netdata only. With the current config, anyone on the local host can get nginx stats